services/provider/proto/provider.proto

```diff
@@ -171,6 +171,8 @@ message ReportStatusRequest{
     string clusterName = 5;
     // clientName is the name of the connected storageclient
     string clientName = 6;
+    // storageQuotaUtilizationRatio is the ratio of utilized quota of connected client
+    string storageQuotaUtilizationRatio = 7;
```

Comment:
Why a string? we want a ratio which means a floating point number

Reply:
The reason for choosing the string for this is i saw the platform version is string even when its floating so i made it string. Now i checked its values and saw it has multiple decimal points in it(like `4.16.0 / 4.17.0`). I will change to `float32`